### PR TITLE
S1 api pull

### PR DESCRIPTION
**Pull request by Benson-J**
Scrapes data from api & adds it to the database.

To be reviewed:
dataScrape/apiPull.php
dataScrape/src/apiPull.php
dataScrape/src/pokemonTableUpdate.php

### CODE REVIEW

File: dataScrape/src/pokemonTableUpdate.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    private $pokemonDataList;
+
+    /**
+     * pokemonTableUpdate constructor.  Opens pokedex db connection
+     *
+     * @param array $pokemonDataList contains an array for each pokemon
+     */
+    public function __construct(array $pokemonDataList)
+    {
+        $this->db = new PDO('mysql:dbname=pokedex_hb;host=127.0.0.1', 'root');
```
Comment (hoptond) [inline dataScrape/src/pokemonTableUpdate.php]: Does not work on machines with virtualbox.

File: dataScrape/src/pokemonTableUpdate.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?php
+
```
Comment (hoptond) [inline dataScrape/src/pokemonTableUpdate.php:2]: Is this being namespaced? I'm fairly certain everything else in our project will be namespaced so this should be as well.

File: dataScrape/apiPull.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+require_once 'src/apiPull.php';
+require_once 'src/pokemonTableUpdate.php';
```
Comment (hoptond) [inline dataScrape/apiPull.php:4]: Could replace these two with an autoloader.

File: dataScrape/src/apiPull.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                CURLOPT_HTTPHEADER => array(
+                    "cache-control: no-cache"
+                ),
+            ));
+
+            $response = curl_exec($curl);
+
+            curl_close($curl);
+
+            $pokemon = json_decode($response);
```
Comment (hoptond) [inline dataScrape/src/apiPull.php:31]: Is there a way of just getting back the information we need, and not all the information available from the API? If there isn't don't worry about it.

**Review by hoptond — CHANGES_REQUESTED** [review]

File: dataScrape/src/apiPull.php (hunk 4 above)
Comment (Benson-J) [reply dataScrape/src/apiPull.php:31]: Not possible without doing multiple requests per pokemon, which would hit the api request limit too soon.

File: dataScrape/src/pokemonTableUpdate.php (hunk 1 above)
Comment (Benson-J) [reply dataScrape/src/pokemonTableUpdate.php]: Addressed in b642512.

File: dataScrape/src/pokemonTableUpdate.php (hunk 2 above)
Comment (Benson-J) [reply dataScrape/src/pokemonTableUpdate.php:2]: This part of the application is standalone, so does not need to be namespaced.

File: dataScrape/apiPull.php (hunk 3 above)
Comment (Benson-J) [reply dataScrape/apiPull.php:4]: This part of the application is standalone, so does not need to be autoloaded.

**Review by hoptond — APPROVED** [review]

**Pull request merged**